### PR TITLE
Add CANFundingDetails Endpoint

### PR DESCRIPTION
**Pull request by rajohnson90**
## What changed

Adding the /can-funding-details/ and /can-funding-details/<id> endpoints. This api endpoint has no consumers yet so there are no front-end interactions to be concerned with. This change will allow a user to get can-funding-details objects as well as create, update, and delete them. 

## Issue

[#2838 ](https://github.com/HHS/OPRE-OPS/issues/2838)

## How to test

1. Run python unit tests

## Definition of Done Checklist
- [x] OESA: Code refactored for clarity
- [x] OESA: Dependency rules followed
- [x] Automated unit tests updated and passed
- [x] Automated integration tests updated and passed
- [x] Automated quality tests updated and passed
- [x] Automated load tests updated and passed
- [x] Automated a11y tests updated and passed
- [x] Automated security tests updated and passed
- [x] 90%+ Code coverage achieved
- [x] Form validations updated

### CODE REVIEW

File: backend/ops_api/ops/schemas/cans.py [hunk 1]
```diff
@@ -101,16 +101,30 @@ class CreateUpdateFundingBudgetSchema(Schema):
     notes = fields.String(load_default=None)
 
 
+class CreateUpdateFundingDetailsSchema(Schema):
```
Comment (fpigeonjr) [inline backend/ops_api/ops/schemas/cans.py:104]: are there any openApi schema changes needed?

**Review by fpigeonjr — COMMENTED** [review]
- will defer to others

File: backend/ops_api/ops/schemas/cans.py (hunk 1 above)
Comment (rajohnson90) [reply backend/ops_api/ops/schemas/cans.py:104]: There are, Frank. Good callout. I will get these.

**Review by maiyerlee — APPROVED** [review]
👏👏👏

File: backend/ops_api/ops/schemas/cans.py (hunk 1 above)
Comment (rajohnson90) [reply backend/ops_api/ops/schemas/cans.py:104]: Just fixed this by adding the endpoints and types.

Comment (fpigeonjr) [reply backend/ops_api/ops/schemas/cans.py:104]: thx @rajohnson90

**Review by johndeange — APPROVED** [review]

**Pull request merged**